lib/matplotlib/text.py

```diff
@@ -176,9 +176,11 @@ def update(self, kwargs):
         kwargs = dict(kwargs)
         sentinel = object()  # bbox can be None, so use another sentinel.
         # Update fontproperties first, as it has lowest priority.
-        fontproperties = kwargs.pop("fontproperties", sentinel)
-        if fontproperties is not sentinel:
-            self.set_fontproperties(fontproperties)
+        alias_map = ["fontproperties", *self._alias_map["fontproperties"]]
+        for alias in alias_map:
+            fontproperties = kwargs.pop(alias, sentinel)
+            if fontproperties is not sentinel:
+                self.set_fontproperties(fontproperties)
```

Comment:
I can't fully comprehend how this will impact the system, so I made a brief investigation and some tests involving `bboxes` and it seems nothing is broken. Also, the tests are passing. 

If someone more experienced than me approves this I would be fine with that.

Reply:
Is this a case for https://matplotlib.org/devdocs/api/cbook_api.html#matplotlib.cbook.normalize_kwargs

Reply:
> Is this a case for https://matplotlib.org/devdocs/api/cbook_api.html#matplotlib.cbook.normalize_kwargs

Could you explain more about that? I'm not quite familiar with `cbook` actually. Maybe more elegant approach exists.

Reply:
Yes, I think this should use `normalize_kwargs`. See examples in `lib/matplotlib/axes/_axes.py`